### PR TITLE
add cross-links, little rewording and structure revision

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary
- add cross link on the prerequisite
- little rewording
- reorganized heading for update status
- add a harvest job configuration section

### CODE REVIEW

**Review by nutjob4life — APPROVED** [review]
Looks good

**Review by tdddblog — APPROVED** [review]

**Comment by tdddblog** [comment]
I approved it, but we have to discuss what is the best way to handle cross references in Sphinx.

File: docs/source/admin/create_reg.rst [hunk 1]
```diff
@@ -22,8 +23,8 @@ The indices are created with Registry Manager command-line tool.
 Prerequisites
 *************
 
- * OpenSearch server is running.
- * Registry Manager command-line tool is installed.
+ * OpenSearch server is `running <https://opensearch.org/>`_.
+ * Registry Manager command-line tool is `installed <../install/tools.html#registry-manager>`_.
```
[the remark this answers is not in the document]
Comment (jordanpadams) [reply docs/source/admin/create_reg.rst:27]: @tdddblog agreed that is the "ideal" syntax but we use these inline links all over the place.

@tloubrieu-jpl take note above for the future

File: docs/source/user/delete_data.rst [hunk 2]
```diff
@@ -80,7 +80,8 @@ and provide following parameters:
 OpenSearch API
 *****************
 
-You can also use OpenSearch 
+**Warning:** only use if you know what you are doing
```
[the remark this answers is not in the document]
Comment (jordanpadams) [reply docs/source/user/delete_data.rst:83]: @tloubrieu-jpl +1 here. can we make this quick change?

**Review by jordanpadams — CHANGES_REQUESTED** [review]

File: docs/source/user/delete_data.rst [hunk 3]
```diff
@@ -80,7 +80,9 @@ and provide following parameters:
 OpenSearch API
 *****************
 
-You can also use OpenSearch 
+.. warning::
```
Comment (jordanpadams) [inline docs/source/user/delete_data.rst:83]: @tloubrieu-jpl @tdddblog updated

**Review by jordanpadams — APPROVED** [review]

**Comment by tloubrieu-jpl** [comment]
@tdddblog @jordanpadams thanks for the feedback and correcting this warning. I was lazy for the cross-reference links...

**Pull request merged**